eric_sse/patterns.py

```
@@ -0,0 +1,18 @@
+from eric_sse.handlers import QueuingErrorHandler
+from eric_sse.message import MessageContract
+from eric_sse.queues import Queue
+from eric_sse import get_logger
+
+logger = get_logger()
+
+class DeadLetterQueueHandler(QueuingErrorHandler):
+    def __init__(self, queue: Queue):
+        self.__queue = queue
+
+    def handle_push_error(self, msg: MessageContract, exception: Exception):
```

Ruff (0.12.2)

Unused method argument: `exception`

(ARG002)